ext/debug/extconf.rb

```diff
@@ -4,6 +4,7 @@
 $distcleanfiles << "debug_version.h"
 
 if defined? RubyVM
+  $defs << '-DCRUBY'
```

Comment:
No C macros to determine CRuby or others?

Reply:
I'm not aware of one.
We could be more specific and name it `HAVE_RB_ISEQ` or so maybe because it's all about iseq, but that may be confusing given we don't use `have_*` to check that.
Is `rb_iseq` exposed in CRuby headers?

Reply:
`RUBY_RUBY_H` for example? (`include/ruby/ruby.h`)

Reply:
TruffleRuby also has ruby/ruby.h and reuses CRuby headers: https://github.com/oracle/truffleruby/tree/master/lib/cext/include, so that wouldn't be a good term for it:

Reply:
I changed it to `HAVE_RB_ISEQ`

Reply:
Another `RB_ISEQ_...` means Ruby has corresponding functions, so `HAVE_RB_ISEQ` means having `rb_iseq()`.
So... anyway, I'll change it later.